### ISSUE TITLE
`fs.copyFile()` does not work with `/snapshot` paths

### ISSUE THREAD

**Issue by TooTallNate** [vercel/pkg#639]
https://nodejs.org/api/fs.html#fs_fs_copyfile_src_dest_flags_callback

Example error:

```
{ Error: ENOENT: no such file or directory, copyfile '/snapshot/foo/bar/index.js.map' -> '/Cache/foo/bar/index.js.map'
  errno: -2,
  code: 'ENOENT',
  syscall: 'copyfile',
  path: '/snapshot/foo/bar/index.js.map',
  dest: '/Cache/foo/bar/index.js.map' }
```

**Comment by sorleone** [comment]
I too have this bug and this is a temporary solution that works:

`writeFileSync(path, readFileSync(src), { flag: 'wx' });`

**Comment by jingsam** [comment]
I encounter same issue when use `fs-extra` `copySync` function. pkg `4.2.6` work fine.

**Comment by timsuchanek** [comment]
We have the same issue for the `prisma` CLI.

**Comment by jingsam** [comment]
ops！ This issue does't resolve yet

**Comment by kamesh95** [comment]
@TooTallNate  Is there an update on this issue? I am still facing this in pkg v5.2.1

**Comment by kamesh95** [comment]
The same thing happens when using chmod function as well. It says:
```
ENOENT: no such file or directory, chmod [[PATH_TO_FILE]]
```

I checked the path using exists function and it returned true that file does exist. But in chmod the same path fails as ENOENT
```
console.warn("exists:::", fse.existsSync(SCRIPT_PATH));
fse.chmodSync(SCRIPT_PATH, "755");
```
Output of above lines:
```
exists::: true
Exception:  
ENOENT: no such file or directory, chmod [[SCRIPT_PATH]]
```

**Comment by jingsam** [comment]
bump

**Comment by robertsLando** [comment]
It could be because it's missing here: https://github.com/vercel/pkg/blob/main/prelude/bootstrap.js#L502

**Comment by jingsam** [comment]
.

**Comment by robertsLando** [comment]
@jingsam PR?

**Comment by AzazelloB** [comment]
The workaround

```
async function copy(source, target) {
    // use stream pipe to reduce memory usage
    // when loading a large file into memory.
    const writeFileStream = fs.createWriteStream(target);
    const readFileStream = fs.createReadStream(source).pipe(writeFileStream);

    return new Promise(function(resolve, reject) {
        writeFileStream.on('finish', resolve);
        readFileStream.on('error', reject);
    });
}
```

**Comment by AzazelloB** [comment]
@jingsam ^

**Comment by robertsLando** [comment]
I already opened a pr to fix this: #1484

**Comment by jesec** [comment]
dupe of #420.
